src/main/java/frc/robot/subsystems/VisionSubsystem.java

```diff
@@ -0,0 +1,107 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import frc.robot.RobotConstants;
+
+import org.photonvision.PhotonCamera;
+import org.photonvision.targeting.PhotonTrackedTarget;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Transform3d;
+import edu.wpi.first.util.sendable.SendableBuilder;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import frc.robot.pose.RecognizedAprilTagTarget;
+import frc.robot.pose.VisionPose;
+import org.photonvision.targeting.PhotonPipelineResult;
+
+public class VisionSubsystem extends EntechSubsystem {
+  PhotonTrackedTarget bestTarget;
```

Comment:
these are missing private keywoards. leaving them out makes them 'package protected', which we dont want